pages/hydra/1_new_start_here/get-help.md

```diff
@@ -18,7 +18,7 @@ For technical questions, the first place to reach people is in the Samvera Slack
 
 - [Samvera Slack team](https://project-hydra.slack.com)
 
-Joining the Slack team requires an invitation, but anyone can generate an automatic invite using this [form](http://slack.projecthydra.org/). (If this form does not work, you may want to use this backup [form](https://docs.google.com/forms/d/1cRJaOTo2pHAeMzfE3hKL3toPHF2_e4ZXw3ufOXlF2vU/viewform). If you need help getting connected, ping slack@projecthydra.org and someone will be in touch.
+Joining the Slack team requires an invitation, but anyone can generate an automatic invite using this [form](http://slack.samvera.org/). (If this form does not work, you may want to use this backup [form](https://docs.google.com/forms/d/1cRJaOTo2pHAeMzfE3hKL3toPHF2_e4ZXw3ufOXlF2vU/viewform). If you need help getting connected, ping slack@samvera.org and someone will be in touch.
```

Comment:
hey @jcoyne, the slack samvera url isn't functional yet. Is that change in the works, coming up soon?